src/renderer/OpenGL.zig

```diff
@@ -2298,6 +2437,31 @@ pub fn drawFrame(self: *OpenGL, surface: *apprt.Surface) !void {
         }
     }
 
+    // Check if we need to update our current background image
+    if (self.current_background_image) |current_background_image| {
+        switch (current_background_image) {
+            .ready => {},
+
+            .pending_gray,
+            .pending_gray_alpha,
+            .pending_rgb,
+            .pending_rgba,
+            .replace_gray,
+            .replace_gray_alpha,
+            .replace_rgb,
+            .replace_rgba,
+            => try self.current_background_image.?.upload(self.alloc),
```

Comment:
I still feel I'm a tad allergic to `.?`s. Sorry.

```suggestion
            => try current_background_image.upload(self.alloc),
```

Reply:
Done.